tests/rt.conf

```diff
@@ -185,6 +187,7 @@ RUN     | fv3_ccpp_wrtGauss_nemsio_c192
 RUN     | fv3_ccpp_stochy                                                                                                                | standard    |                | fv3         |
 RUN     | fv3_ccpp_iau                                                                                                                   | standard    |                |             |
 RUN     | fv3_ccpp_lheatstrg                                                                                                             | standard    |                |             |
+RUN	| fv3_ccpp_ca															 | standard    |                | fv3         |
```

Comment:
Note that this entry should not have "fv3" in the last column, because it will overwrite the IPD results in the baseline. Only IPD REPRO or CCPP REPRO should have the "fv3" in the last column". For fv3_stochy, I changed this a while back, that is CCPP REPRO creates the baseline and IPD REPRO is skipped when creating the baseline.